engine/runtime/src/main/java/org/enso/interpreter/runtime/data/ArrayOverBuffer.java

```diff
@@ -38,4 +50,49 @@ long getArraySize() {
   public static ArrayOverBuffer wrapBuffer(ByteBuffer buffer) {
     return new ArrayOverBuffer(buffer);
   }
+
+  @ExportMessage
+  String toDisplayString(boolean allowSideEffects) {
```

Comment:
As below, this is basically copied over from `Vector`. Cannot we have this common logic extracted into some common function and delegated to, instead of duplicating the code? It's quite simple but it's a big chunk of logic here, so duplicating it just seems bad.